### PR TITLE
Update description of mariadb-common package

### PR DESCRIPTION
**Pull request by an3l**
- Commit https://github.com/mariadb/server/commit/438ed0408c69
introduced `mariadb-common` package and added description. Example in
description is confusing since files referred to are not installed and
not related/owned by the `mariadb-common` package.
- Patch is updating real file/directory description section pointing to
the real examples owned by the package.
- Example:
```
$ dpkg --get-selections maria*
mariadb-client-10.3				install
mariadb-client-core-10.3			install
mariadb-common					install
mariadb-server					install
mariadb-server-10.3				install
mariadb-server-core-10.3			install

$ dpkg -L mariadb-common
/.
/etc
/etc/mysql
/etc/mysql/mariadb.cnf
/etc/mysql/mariadb.conf.d
/usr
/usr/share
/usr/share/doc
/usr/share/doc/mariadb-common
/usr/share/doc/mariadb-common/changelog.Debian.gz
/usr/share/doc/mariadb-common/copyright

$ ls /etc/mysql/
conf.d/          debian-start     debian.cnf       mariadb.cnf      mariadb.conf.d/  my.cnf           my.cnf.fallback  

```

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -121,14 +121,14 @@ Description: MariaDB database common files (e.g. /etc/mysql/my.cnf)
...
-Description: MariaDB database common files (e.g. /etc/mysql/conf.d/mariadb.cnf)
+Description: MariaDB database common files (e.g. /etc/mysql/mariadb.conf.d/)
  MariaDB is a fast, stable and true multi-user, multi-threaded SQL database
  server. SQL (Structured Query Language) is the most popular database query
  language in the world. The main goals of MariaDB are speed, robustness and
  ease of use.
  .
  This package includes files needed by all versions of the client library
- (e.g. /etc/mysql/conf.d/mariadb.cnf).
+ (e.g. /etc/mysql/mariadb.conf.d/ or /et/mysql/mariadb.cnf).
```
Comment (grooverdan) [inline debian/control:131]: typo '/et/'

Comment (an3l) [reply debian/control]: Thanks @grooverdan

**Comment by an3l** [comment]
@ottok  any update regarding ^ ?

**Review by ottok — APPROVED** [review]

**Comment by an3l** [comment]
Thanks @ottok for approval. 
I have rebased on 10.2 with patch https://github.com/MariaDB/server/commit/26015fc994dde5aa7db4659ae02ba87ec2dd4ebd
After protected branch passes I will push it to 10.2 since 10.1 is EOL.

**Comment by an3l** [comment]
Pushed to 10.2 with https://github.com/MariaDB/server/commit/85063aebda7e4b691c04a90b0c38a6df3873a84e.
Thanks

**Pull request closed without merging**